src/main/java/iudx/resource/server/apiserver/validation/types/ElasticSearchResponseTypeValidator.java

```diff
@@ -0,0 +1,55 @@
+package iudx.resource.server.apiserver.validation.types;
+
+import iudx.resource.server.apiserver.exceptions.DxRuntimeException;
+import iudx.resource.server.common.HttpStatusCode;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+import static iudx.resource.server.apiserver.util.Constants.*;
+import static iudx.resource.server.common.ResponseUrn.INVALID_HEADER_VALUE_URN;
+import static iudx.resource.server.common.ResponseUrn.INVALID_ID_VALUE_URN;
+
+public class ElasticSearchResponseTypeValidator implements Validator {
```

Comment:
rename class name to ElasticSearchFileResponseTypeValidator

Reply:
Oh okay, changed it here : [Change file name](https://github.com/datakaveri/iudx-resource-server/pull/425/commits/828af3cccd5da00f1615be759e363da623d17d3e)